packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -624,8 +623,8 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
 
   // SCROLL WHEEL
 
-  // Returns the offset that should result from applying [event] to the current
-  // position, taking min/max scroll extent into account.
+  // Returns the offset that should result from applying [event] with axis and
+  // direction taken into account.
   double _targetScrollOffsetForPointerScroll(PointerScrollEvent event) {
```

Comment:
You've changed this from computing the target scroll offset to returning a modified delta; the name and description should be updated accordingly.

Reply:
I've changed it, but for some reason it's not marked Outdated.